### ISSUE TITLE
Workspaces not assigned to group; or switched to, and no icons

### ISSUE THREAD

**Issue by githubforgetmenot** [infokiller/i3-workspace-groups#16]
Output of `i3 --moreversion 2>&- || i3 --version` and `pip show i3pci` and `python3.7 --version`

`Binary i3 version:  4.16.1-184-g00181678 (2019-07-03, branch "shape") © 2009 Michael Stapelberg and contributors
Running i3 version: 4.16.1-184-g00181678 (2019-07-03, branch "shape") (pid 20125)`

`Name: i3ipc
Version: 2.1.1`

`Python 3.7.3`

**What I tried:**

To assign workspaces to groups
Switch to a workspace within a group using the bind (i.e workspace number). 

(also I had to rename all bindcodes to bindsyms before they worked.)

In addition, I tried to exec the commands from the terminal:

```[user@arch bin]$ i3-assign-workspace-to-group
Traceback (most recent call last):
  File "/home/user/.local/bin/i3-workspace-groups", line 198, in <module>
    main()
  File "/home/user/.local/bin/i3-workspace-groups", line 192, in main
    controller.assign_workspace_to_group(args.group)
  File "/home/user/.local/lib/python3.7/site-packages/i3wsgroups/i3_workspace_groups.py", line 604, in assign_workspace_to_group
    self.organize_workspace_groups(group_to_workspaces)
  File "/home/user/.local/lib/python3.7/site-packages/i3wsgroups/i3_workspace_groups.py", line 478, in organize_workspace_groups
    workspace.name, new_name))
  File "/home/user/.local/lib/python3.7/site-packages/i3wsgroups/i3_workspace_groups.py", line 420, in send_i3_command
    if not reply[0]['success']:
TypeError: 'CommandReply' object is not subscriptable

[user@arch bin]$ i3-workspace-groups workspace-number 2
Traceback (most recent call last):
  File "/home/user/.local/bin/i3-workspace-groups", line 198, in <module>
    main()
  File "/home/user/.local/bin/i3-workspace-groups", line 171, in main
    controller.focus_workspace_number(args.workspace_relative_number)
  File "/home/user/.local/lib/python3.7/site-packages/i3wsgroups/i3_workspace_groups.py", line 635, in focus_workspace_number
    target_local_number)
  File "/home/user/.local/lib/python3.7/site-packages/i3wsgroups/i3_workspace_groups.py", line 620, in _get_workspace_name_from_context
    self.organize_workspace_groups(group_to_workspaces)
  File "/home/user/.local/lib/python3.7/site-packages/i3wsgroups/i3_workspace_groups.py", line 478, in organize_workspace_groups
    workspace.name, new_name))
  File "/home/user/.local/lib/python3.7/site-packages/i3wsgroups/i3_workspace_groups.py", line 420, in send_i3_command
    if not reply[0]['success']:
TypeError: 'CommandReply' object is not subscriptable
```

**What I saw:**

Rofi opened, but the current workspace was not assigned to the group(?).
The workspace-number binds did not seem to show any response.

**What I expected:**

Current workspace to be assigned to the group specified in rofi
To switch to (/create) a new workspace within a group, as mentioned in readme: `So you press Super+2, which will move you to a new workspace named "work:2"`

**Final notes**

I am able to (i3-) switch active workspace group, and workspace_prev/next binds move me to the first group, then remains there. Lastly, When I attempted to autoname the workspaces from terminal I just saw some little boxes with numbers in bar. I expect all of this is my total misunderstanding of how your program works, or perhaps it is a version problem?

By the way, I love this concept. The idea, to use an analogy, for example that we would have one folder containing all our files is ridiculous. Clearly there needs to be sub folders (groups).

**Comment by infokiller** [comment]
Thanks a lot for the details report and sorry for the trouble!

As for the error you're seeing, I suspect this is related to the `i3ipc` version. On my machine `1.7.1` is installed, and I guess that version `2.0.0` introduced breaking changes. I'll try to verify this is indeed the issue, but in the meantime can you please try downgrading `i3ipc`? I think you can do this with:
`pip install --force-reinstall i3ipc==1.7.1`?

Also, I'm not entirely sure that `i3-workspace-groups` works with Python 3.7 since I only tested it on 3.6. It should probably work though.

(P.S: indeed, the README mentions some `bindcode` command where it should actually be `bindsym`, I fixed that now)

**Comment by infokiller** [comment]
I confirmed this issue is related to `i3ipc` versions `>2`. This is fixed in `i3_workspace_groups` version `0.3.4` (which pins `i3ipc` to the latest version before 2) and version `0.4.1-alpha` (which contains a minor update to be compatible with `i3ipc` version 2).
So to fix this you can now run: `pip install --force-reinstall i3-workspace-groups`.

Closing this, but please reopen if the fix doesn't work for you.
